.github/workflows/deploy.yml

```diff
@@ -136,7 +136,7 @@ jobs:
       
       - name: Run patcher
         run: |
-          curl -L -o patcher https://github.com/amsyarasyiq/bunny-ipa-patcher/releases/download/release-pyon/patcher.mac-amd64
+          curl -L -o patcher https://github.com/Cristiandis/ipa-patcher/releases/download/release-pyon/patcher.mac-amd64
```

Comment:
Please use a specific release tag and a hash check in this step to ensure the binary doesn't change.